### PR TITLE
feat(web): add feedback widget

### PR DESCRIPTION
**Pull request by 0xGabi**
Solve #168 

Screenshot 2023-09-27 at 1 03 35 PM

### CODE REVIEW

File: apps/web/src/pages/api/feedback.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-case-declarations */
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const webhooks = [
+  process.env.FEEDBACK_WEBHOOK_URL ??
+    "https://discord.com/api/webhooks/abc/123",
```
Comment (PabloCastellano) [inline apps/web/src/pages/api/feedback.ts:6]: This is not a valid default so I'd rather remove it and make the variable optional

File: apps/web/src/pages/api/feedback.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const { message, rate, metadata } = req.body;
+  const method = req.method;
+
+  try {
+    switch (method) {
+      case "POST":
+        const text = `New Feedback ✨
+
+Message: ${message ? message : "-"}
+Rate: ${rate ? rateEmoji.get(rate) : "-"}
```
Comment (PabloCastellano) [inline apps/web/src/pages/api/feedback.ts:25]: Should we make 'rate' mandatory?

**Review by PabloCastellano — COMMENTED** [review]
I like the simplicity but have one concern, can we somehow rate-limit this endpoint to avoid spam?

File: turbo.json [hunk 3]
```diff
@@ -51,6 +51,7 @@
     "CHAIN_ID",
     "CI",
     "DATABASE_URL",
+    "FEEDBACK_WEBHOOK_URL",
```
Comment (PabloCastellano) [inline turbo.json:54]: Documentation should be updated too

File: apps/web/src/pages/api/feedback.ts (hunk 2 above)
Comment (0xGabi) [reply apps/web/src/pages/api/feedback.ts:25]: This needs to be configured on the component. And there is no configuration for that allowed. To include this we should fork and make our own version of the widget.

**Pull request merged**